docs/src/api/class-browsercontext.md

```diff
@@ -1025,6 +1025,37 @@ handler function to route the request.
 
 How often a route should be used. By default it will be used every time.
 
+## async method: BrowserContext.routeFromHar
+
+Provides the capability to serve network requests that are made in the context from prerecorded HAR file.
+
+If HAR file contains an entry with the matching url and HTTP method, then the entry's headers, status and body will be used to fulfill. An entry resulting in a redirect will be followed automatically. If there is no matching entry in the file the execution continues to try other configured HAR files and [Route] handlers.
+If `path` is a relative path, then it is resolved relative to the current working directory.
+
+:::note
+[`method: BrowserContext.routeFromHar`] will not intercept requests intercepted by Service Worker. See [this](https://github.com/microsoft/playwright/issues/1090) issue. We recommend disabling Service Workers when using request interception. Via `await context.addInitScript(() => delete window.navigator.serviceWorker);`
+:::
+
+### param: BrowserContext.routeFromHar.harPath
+- `harPath` <[path]>
+
+Path to the HAR file with prerecorded network data.
+
+### option: BrowserContext.routeFromHar.strict
+- `strict` <[boolean]>
+
+If set to true any request not found in the HAR file will be aborted. If set to
+false missing requests will continue normal flow and can be handled by other
+[Route] handlers or served from other HAR files configured with [`method: BrowserContext.routeFromHar`].
+Defaults to false.
```

Comment:
I thought we agreed to default to `true`?

Reply:
Let's double check tomorrow in the meeting.